lib/tasks/production-copy.rake

```diff
@@ -264,23 +262,23 @@ module CloneProductionHelper
 
   def balanced_payments_refs_to_clear
     if ENV["KEEP_BALANCED_URIS"] =~ /^(y|on|true)/i
-      [ { model: Payment,     fields: [ :balanced_uri ] } ]
+      [ { model: ::Payment,     fields: [ :balanced_uri ] } ]
```

Comment:
ensures proper namespacing.. was getting error _"Could not find CloneProductionHelper::Payment"_